src/test/java/codeu/controller/ChatServletTest.java

```diff
@@ -135,7 +135,7 @@ public void testDoPost_ConversationNotFound() throws IOException, ServletExcepti
     Mockito.when(mockRequest.getRequestURI()).thenReturn("/chat/test_conversation");
     Mockito.when(mockSession.getAttribute("user")).thenReturn("test_username");
 
-    User fakeUser = new User(UUID.randomUUID(), "test_username", Instant.now());
+    User fakeUser = new User(UUID.randomUUID(), "test_username", "password",Instant.now());
```

Comment:
add a space after "password", (here and other places)

Reply:
Done